demo-shell-ng2/app/components/files/files.component.css

```diff
@@ -8,6 +8,10 @@
     }
 }
 
+.files-toolbar-title {
+    max-width: calc(100% - 160px);
```

Comment:
Still, does not work for the flex-based layout. We'll need to find out some other solution.